extensions/wikia/AdEngine/js/lookup/rubicon/rubiconFastlane.js

```diff
@@ -173,6 +173,22 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconFastlane', [
 			}
 		});
 	}
+	
+	function shouldHandleFloorPrice(floorPrice, slotName, rubiconTierKeyParam) {
+		return typeof floorPrice !== 'undefined' &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)